airflow/ui/src/components/Time.tsx

```diff
@@ -34,9 +34,10 @@ dayjs.extend(advancedFormat);
 type Props = {
   readonly datetime?: string | null;
   readonly format?: string;
+  readonly tooltip?: boolean;
```

Comment:
```suggestion
  readonly showTooltip?: boolean;
```

Let's add a verb since we're not actually passing a tooltip but a boolean